data/json/items/gun/380.json

```
@@ -171,5 +171,55 @@
     "flags": [ "RELOAD_EJECT", "EASY_CLEAN" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "380": 1 } } ],
     "melee_damage": { "bash": 10 }
+  },
+  {
+    "id": "hpt3895",
+    "looks_like": "cx4",
+    "type": "GUN",
+    "name": { "str": ".380 ACP pistol-caliber carbine" },
+    "description": "A pistol-caliber carbine chambered in .380 ACP.  It is more powerful and accurate than an equivalent pistol but the .380 ACP round is nonetheless fairly weak.  Accepts 10-round magazines, which are not shared with any .380 ACP pistol unlike many other similar pistol-caliber carbines.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "hpt3895",
+        "name": { "str": "Hi-Point CF3895" },
+        "description": "A pistol-caliber carbine chambered in .380 ACP, designed by Hi-Point Firearms.  Its 16.5\" barrel provides extra stopping power over an equivalent pistol, but .380 is still a relatively weak caliber.  Accepts 10-round magazines, which are notably not interchangeable with those of CF380 pistols.",
+        "weight": 1
+      }
+    ],
+    "weight": "2835 g",
+    "volume": "1881 ml",
+    "longest_side": "787 mm",
+    "price": 31500,
+    "price_postapoc": 2500,
+    "to_hit": -1,
+    "symbol": "(",
+    "material": [ "plastic", "steel", "zinc" ],
+    "color": "dark_gray",
+    "ammo": [ "380" ],
+    "skill": "rifle",
+    "range": 17,
+    "dispersion": 180,
+    "durability": 8,
+    "blackpowder_tolerance": 30,
+    "min_cycle_recoil": 270,
+    "ranged_damage": { "damage_type": "bullet", "amount": 2 },
+    "valid_mod_locations": [
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail", 1 ],
```

```suggestion
      [ "rail mount", 1 ],
```
The 3895 has bottom and top rails, but not side ones.

---

thanks, I just sorta copypasted mod options. I'll definitely be doing more of these.